sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -123,6 +319,63 @@ export interface EntityRecognitionSuccessResult extends TextAnalysisSuccessResul
     readonly entities: Entity[];
 }
 
+// @public
+export type FhirVersion = string;
+
+// @public
+export type HealthcareAction = ActionPrebuilt & {
+    setFhirVersion?: FhirVersion;
+    stringIndexType?: StringIndexType;
+};
+
+// @public
+export interface HealthcareAssertion {
+    association?: Association;
+    certainty?: Certainty;
+    conditionality?: Conditionality;
+}
+
+// @public
+export interface HealthcareEntity extends Entity {
+    assertion?: HealthcareAssertion;
+    category: HealthcareEntityCategory;
+    dataSources: EntityDataSource[];
+    normalizedText?: string;
+}
+
+// @public
+export type HealthcareEntityCategory = string;
+
+// @public
+export interface HealthcareEntityRelation {
+    relationType: RelationType;
+    roles: HealthcareEntityRelationRole[];
+}
+
+// @public
+export interface HealthcareEntityRelationRole {
+    entity: HealthcareEntity;
+    name: HealthcareEntityRelationRoleType;
+}
+
+// @public
+export type HealthcareEntityRelationRoleType = string;
+
+// @public
+export type HealthcareErrorResult = TextAnalysisErrorResult;
+
+// @public
+export type HealthcareResult = HealthcareSuccessResult | HealthcareErrorResult;
+
+// @public
+export interface HealthcareSuccessResult extends TextAnalysisSuccessResult {
+    entities: HealthcareEntity[];
+    entityRelations: HealthcareEntityRelation[];
+    fhirBundle?: {
+        [propertyName: string]: any;
```

Comment:
This is auto-generated but I think that we don't expect the customer to cast this to a concrete type since no such type is provided at the moment and the schema is complicated enough to be typed by the customer.

That being said, I would like to revisit this after the first beta release and see what can we do here.